sdk/python/kfp/compiler/compiler.py

```diff
@@ -832,8 +832,8 @@ def _create_pipeline_workflow(self,
 
         # set ttl after workflow finishes
         if pipeline_conf.ttl_seconds_after_finished >= 0:
-            workflow['spec'][
-                'ttlSecondsAfterFinished'] = pipeline_conf.ttl_seconds_after_finished
```

Comment:
This seems to be a breaking change, do we know from which Argo version was this change introduced? Consequently, which KFP version has that Argo upgrade? We'll need to document this change in the [release note](https://github.com/kubeflow/pipelines/blob/master/sdk/RELEASE.md).